apps/web/src/components/providers/convex-client-provider.tsx

```
@@ -28,6 +30,17 @@ export default function ConvexClientProvider({
 }: {
   children: ReactNode;
 }) {
+  const posthog = usePostHog();
+  const session = useAuth();
+
+  // track users by id in posthog
+  useEffect(() => {
+    if (session && session.userId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)